gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -76,6 +83,27 @@
   public boolean autoCheckForUpdates = true;
   public boolean updateAvailable = false;
   public long lastCheckForUpdates = 0; // milliseconds since midnight, January 1, 1970 UTC.
+  public String analyticsClientId = ""; // Empty means do not track.
+
+  /**
+   * listen registers the listener for changes.
+   */
+  public void listen(Listener listener) {
```

Comment:
Please call this addListener and add a removeListener method.

Reply:
Done.